api/v1beta1/machinedeployment_types.go

```diff
@@ -211,6 +216,31 @@ type MachineRollingUpdateDeployment struct {
 
 // ANCHOR_END: MachineRollingUpdateDeployment
 
+// ANCHOR: RemediationStrategy
+
+// RemediationStrategy allows to define how the MachineSet can control scaling operations.
+type RemediationStrategy struct {
+	// MaxInFlight determines how many in flight remediations the MachineSet should issue at
+	// any given point in time.
```

Comment:
Just to clarify slightly. It's not about how many new remediations it issues at a given point in time (one reconcile), it's an upper bound of how many can be in flight at the same time?